### PR TITLE
Update Swift icon generation to output properties as members.

### PR DESCRIPTION
**Pull request by pixlwave**
Rather than manually [mapping the icons in compound-ios](https://github.com/element-hq/compound-ios/blob/main/Sources/Compound/Icons/CompoundIcons.swift) this change creates them in a format that is directly usable for us. Part of https://github.com/element-hq/element-x-ios/issues/2101

The first commit contains all the changes to the generator, the second contains the updated assets.

We should do similar for colours too, although we would need https://github.com/element-hq/compound/issues/93 fixed before that is possible.

### CODE REVIEW

File: src/formats/templates/swift/class-members.template [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<%
+//
+// Copyright 2019 Alaska Air Group or its affiliates. All Rights Reserved.
```
Comment (robintown) [inline src/formats/templates/swift/class-members.template:3]: Is this intentional? (this file is actually code from another org?)

File: src/transforms/swift/iconTICamel.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+export function unescape(tokenName: string) {
+  if (tokenName.includes("`")) {
+    return tokenName.replaceAll("`", "")
+  } else {
+    return tokenName
+  }
+}
+
+const swiftKeywords = [
```
Comment (robintown) [inline src/transforms/swift/iconTICamel.ts:55]: Could you add a link to where you've sourced these from?

**Review by robintown — APPROVED** [review]
Looks reasonable :+1:

File: src/formats/templates/swift/class-members.template (hunk 1 above)
Comment (pixlwave) [reply src/formats/templates/swift/class-members.template:3]: Ah yeah I should add a comment that this is based on [any.swift.template](https://github.com/amzn/style-dictionary/blob/main/lib/common/templates/ios-swift/any.swift.template) from the style dictionary source with a small tweak to remove the `static` keyword.

Germain actually opened a [PR](https://github.com/amzn/style-dictionary/pull/972) to handle this upstream but so far it hasn't been reviewed.

File: src/transforms/swift/iconTICamel.ts (hunk 2 above)
Comment (pixlwave) [reply src/transforms/swift/iconTICamel.ts:59]: Updated and added some more that I missed (in reality there is one icon called `public` that caused issues but seemed worthwhile to handle as many as possible).

**Pull request merged**